src/store/constants.js

```diff
@@ -0,0 +1,6 @@
+export const  constants = {
```

Comment:
Maybe add a comment describing what this is for future cohorts, and current team.

Reply:
Added this ✔